### PR TITLE
Enhancement/add toggle firstrun script to sync octoprint credentials

### PR DESCRIPTION
**Pull request by eliasbakken**

### CODE REVIEW

File: roles/install_toggle/tasks/recore.yml [hunk 1]
```diff
@@ -76,3 +76,16 @@
...
+    dest: /etc/systemd/system/toggle-runfirst.service
+
+- name: Recore - Script - Ensure ethernet comes up
+  copy:
+    src: '{{ role_path }}/files/toggle-runfirst'
+    dest: /usr/lib/
+    mode: +x
+
+- name: enable the toggle-runfirst service
+  shell: systemctl enable toggle-runfirst
```
Comment (goeland86) [inline roles/install_toggle/tasks/recore.yml:91]: You can use the systemd ansible module. You'll need to specify that it also runs the daemon-reload, or out won't pick up the new systemd services you added.

**Review by goeland86 — CHANGES_REQUESTED** [review]
Please avoid using the shell module as much as possible.

For one thing the copying of a systemd script without running `systemctl daemon-reload` means that your attempt at enabling a new service will fail.

File: roles/install_toggle/tasks/recore.yml (hunk 1 above)
Comment (eliasbakken) [reply roles/install_toggle/tasks/recore.yml:91]: We have discussed this before, the systemd module does not seem to work when running the INSTALL-*.yml scripts. I would like to use the systemd module, but as far as I can tell there is a bug causing it to not work for me.

Comment (eliasbakken) [reply roles/install_toggle/tasks/recore.yml:91]: Maybe there is a way to separate the two scenarios? I understand that you probably don't want to spend time investigating this, but running INSTALL-toggle.yml behaves differently than build-image-in-chroot-end-to-end.sh

Comment (goeland86) [reply roles/install_toggle/tasks/recore.yml:91]: Yes, and they should.

The ansible scripts we want to use during image build are the SYSTEM-*.yml

Comment (goeland86) [reply roles/install_toggle/tasks/recore.yml:90]: The install scripts are meant to run off you already run on the final board...

Comment (eliasbakken) [reply roles/install_toggle/tasks/recore.yml]: I've added a rescue block, we can try that.

**Pull request merged**